src/Helpers.cxx

```diff
@@ -360,7 +362,7 @@ EVENT::Track* ACTS2Marlin_track(
           track->trackStates().push_back(trackStateAtCalo);
         }
         else{
-          std::cout << "Failed propagation! " << std::endl;
+          streamlog_out(MESSAGE) << "Failed propagation! " << std::endl;
```

Comment:
Our events are dominated by fake seeds from BIB, so we would get this message a few thousand times per event

Reply:
Right, in that case `DEBUG` might be the better option since you probably don't want to produce GBs of logfiles. (However in that case it should be pointed out that the possibility of turning that on at runtime depends on how Marlin was built, so it might not be easily possible to make these messages appear).

Reply:
Done!